### ISSUE TITLE
Open Source Friday - Uwazi - 06-14-24

### ISSUE THREAD

**Issue by vleiloerie** [githubevents/open-source-friday#62]
### Name

Rafael Pólit

### GitHub Handle

@RafaPolit

### Project Name

Uwazi

### Project Repo

https://github.com/huridocs/uwazi

### Stream Date

- [X] Yes
- [ ] Not yet

### Dates

14-06-2024

### Twitter URL

@huridocs

### LinkedIn URL

https://www.linkedin.com/company/huridocs/

### Additional Information

[Uwazi](https://huridocs.org/) is an open-source application that allows you to organise a collection of information and, in the process, reveal previously hidden patterns, relationships and opportunities for action. It is developed by [HURIDOCS](https://huridocs.org/) (Human Rights Information and Documentation Systems), a Geneva-based NGO that, for the past four decades, has equipped human rights defenders with tools and strategies to mobilise information.

In Swahili, Uwazi means “openness,” and it was based on this principle that Uwazi was born. Human rights groups deal with enormous amounts of data, accumulating years’ or even decades’ worth of facts and materials that support the case for change. However, without safe, efficient and collaborative ways of managing it all, many of the insights contained in these collections will remain closed off to the very people who need them most.

Uwazi was first launched in 2017 as a tool to improve access to regional jurisprudence. In consultations with human rights defenders, we had learned that it was difficult for them to not only locate a relevant court ruling, but to also see how it fit into the bigger picture: case documents needed to be connected to the judgment document, which needed to be connected to the deciding judge’s case history, and so on. We thus designed Uwazi as a web-based application for capturing and organising a collection of documents beyond simply its metadata (such as title, author, date published, etc.). Within the substance of a document lies important information and Uwazi enables you to highlight, tag, organise, and relate items to other documents.

But it soon became clear that Uwazi had potential to be much more than just a document management tool. Human rights defenders don’t just work with collections of PDFs and text documents—they also gather troves of information, observations, geolocation points, multimedia files and other data that can be hard to parse in bulk.

We have since expanded Uwazi, creating a database application that is helpful for a wide selection of changemakers, from activists, advocates, lawyers and policymakers who conduct human rights work, to academics, archivists, media organisations and others who work to make a positive difference in the world. 

Initial Uwazi development received significant financial support from the MacArthur Foundation and the Center for Justice and International Law [(CEJIL)](https://uwazi.io/en/entity/tjzuc3a02c). More recently, Uwazi received funding from Google.org to develop its machine learning features to help human rights defenders manage and curate their information collections more effectively. Uwazi continues to be updated and refined regularly by the team at HURIDOCS, which is funded through a combination of grants, payments for services and individual donations.

**Comment by AndreaGriffiths11** [comment]
Hello @vleiloerie! We are excited to host you. To confirm the stream date would be Friday, June, 14 2024. 1PM NY | 5PM UTC

**Comment by vleiloerie** [comment]
Thanks @AndreaGriffiths11, this is well noted!

**Comment by vleiloerie** [comment]
Hey @AndreaGriffiths11 and @LadyKerr,
Our speaker for this event is now confirmed. Please note that our Tech Lead @RafaPolit will be joining you to chat about Uwazi on the 14th of June.   

Thank you!
Yolanda

**Comment by AndreaGriffiths11** [comment]
Hi @vleiloerie would you be so kind to email intro myself (andreagriffiths11@github.com) and @rafapolit to get the calendar invite set. TYSM!

**Comment by AndreaGriffiths11** [comment]
Complete! 
Link to recording: [YouTube](https://www.youtube.com/watch?v=uj4CKNXAm_o&list=PL0lo9MOBetEGqkS7f5mWaFimGRXKTgf4n)
